readthedocs/core/static-src/core/js/doc-embed/rtd-data.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * This exposes data injected during the RTD build into the template. It's
+ * provided via the global READTHEDOCS_DATA variable and is exposed here as a
+ * module for cleaner usage.
+ */
+
+
+var data = READTHEDOCS_DATA;
```

Comment:
This will break on themes that include our JS before the page injected variable here. We need to figure out a better way to keep this state.